src/typeahead/typeahead.directive.ts

```diff
@@ -73,8 +73,11 @@ export class TypeaheadDirective implements OnInit, OnDestroy {
   /** should be used only in case typeaheadSingleWords attribute is true.
    * Sets the word delimiter to match exact phrase.
    * Defaults to simple and double quotes.
-   */
+   */  
   @Input() typeaheadPhraseDelimiters = '\'"';
+  /** used to enable/disable the selection on TAB Event.   
+   */
+  @Input() typeaheadSelectOnTab: boolean = true;
```

Comment:
Hey, thanks for PR, all looks good.
But one change is must for merge,
in terms of app consistency and perf, it would be better to have `selectOnTab` in a config
and not in `@Input`

Reply:
Yes you are right. But same goes for "typeaheadSingleWords" and other boolean values... <br/>
Still: I'm trying to change the `selectOnTab` to be a config and will update my PR.

Reply:
@valorkin @IlyaSurmay I still need a little advice on how to solve this one as a config. Is there an example on how to put things like this in a config?